docs/java/features/odata/use-vdm.md

```diff
@@ -101,6 +101,25 @@ Request parameters:
    - `ignoreAnyVersionIdentifier()` will instead always send a `*` which acts as a wildcard to match all ETags.
 - All operations allow for adding custom headers via `withheader(...)`
 
+Navigable entities:
```

Comment:
We actually thought of having a new section (or even new document) `Advanced OData Features`. because we still have batch, function import, (actions) etc.

And this feels like a  more advanced feature to me, so putting it directly under `Basic CRUD` is maybe not the right place. But we can move it also later on.

Reply:
I'm also in favor of _advanced feature_ section. Although "advanced" would be an exaggeration, we simply mean _non-CRUD_.

However this feature just allows CRUD operations, but on navigation properties instead of root entity-sets. That's why I felt okay with leaving it there.

Reply:
Mhm yes, so from the CRUD point of view it fits there. Maybe we just name the current section `Basic Operations` and then another `Advanced Operations`?

Reply:
I moved the content to a dedicated section within the page.
I wouldn't feel comfortable with introducing a generic "Advanced Operations" headline.